docs/user-guide/resources-properties.md

```diff
@@ -44,30 +44,42 @@ How to manage users and groups is a topic present in the [Managing Users](managi
 
 ## Details
 
-Only for map type resource, and only from the **Edit properties** button <img src="../img/button/edit-icon.jpg" class="ms-docbutton"/> in [Homepage](https://mapstore.geo-solutions.it/mapstore/#/), it is possible to add some details to the map. In this case the *Edit properties* window is the following:
+Only for map type resource, it is possible to add some details to the map. In this case the *Edit properties* window is the following:
```

Comment:
Only resources of type map, it is possible to add details to the map. This is useful to associate some information to the map or an overview description its content. In this case the *Edit properties* window is the following: